Js/localStorage.js

```diff
@@ -1,35 +1,29 @@
 /* eslint-disable linebreak-style */
 /* eslint-disable require-jsdoc */
```

Comment:
- [ ] [OPTIONAL] Kindly make sure not to disable linters, always strive to solve the errors as this helps you write quality code.

Reply:
Ok, awesome @danmainah I will do this going forward! Thank you!